diff --git a/CMakeLists.txt b/CMakeLists.txt
index f654b0e..52b9209 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,7 +109,7 @@ include(policies)
 ################################################################################
 # Check for sqlite3
 ################################################################################
-find_program(EXE_SQLITE3 sqlite3)
+find_program(EXE_SQLITE3 NAMES sqlite3 sqlite3-bin)
 if(NOT EXE_SQLITE3)
   message(SEND_ERROR "sqlite3 binary not found!")
 endif()
