diff --git a/CMakeLists.txt b/CMakeLists.txt
index 617978c..6e6045c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,6 +76,10 @@ SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 # SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${GEOTIFF_BUILD_OUTPUT_DIRECTORY})
 SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 
+###############################################################################
+# PROJ 6.2.0 API change
+add_definitions(-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H)
+
 ###############################################################################
 # Platform and compiler specific settings
 
